velox/functions/lib/aggregates/tests/AggregationTestBase.h

```diff
@@ -206,6 +206,15 @@ class AggregationTestBase : public exec::test::OperatorTestBase {
         });
   }
 
+  /// Use abandon-partial-aggregation optimization to trigger the toIntermediate
```

Comment:
Shouldn't this method be more generic and verify that all logically-equivalent plans fail with the specified error message?

Reply:
I forgot to update the comments. Fixed.